### PR TITLE
Kick off

### PR DESCRIPTION
**Pull request by joredudiaz**
## Summary

- html and scss documents with hello world content

## Jira Story

https://trello.com/c/8F0uTlDl

## Screenshots

image

### CODE REVIEW

File: mock-up/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<!doctype html>
```
Comment (FrankIglesias) [inline mock-up/index.html:1]: check indentation use 2 spaces

File: mock-up/scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.title {
+    color: red
```
Comment (FrankIglesias) [inline mock-up/scss/application.scss]: same

Comment (FrankIglesias) [reply mock-up/scss/application.scss]: missing semi colon

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: mock-up/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet" type="text/css" href="application.css">
```
Comment (betoobernal) [inline mock-up/index.html]: good. But we use 2 spaces for indentation.

**Review by betoobernal — APPROVED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**